airflow/config_templates/config.yml

```diff
@@ -583,11 +583,15 @@ database:
       description: |
         Import path for connect args in SqlAlchemy. Defaults to an empty dict.
         This is useful when you want to configure db engine args that SqlAlchemy won't parse
-        in connection string.
-        See https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_engine.params.connect_args
+        in connection string. This can be set by passing a dictionary containing the create engine parameters. 
+        For more details about passing  create engine parameters(keepalives variables, timeout etc)
+        in postgres db connection see 
+        https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#setting-up-a-postgresql-database
+        Example: connect_args={"timeout":30} can be defined in airflow_local_settings.py and
+        can be imported as shown below
       version_added: 2.3.0
       type: string
-      example: '{"timeout": 30}'
+      example: 'airflow_local_settings.connect_args'
```

Comment:
Initially timeout was defined as 30, should we define the default value of connect_args per that?

Reply:
No, its actually an example.

Reply:
Yeah. The problem here is that you can only add value in the actual "connect_args" which we do not have at this time, so the example shows rather how to configure it not the value to configure.